wiki/Article_Style_Guide/en.md

```diff
@@ -3,28 +3,53 @@
 The Article Style Guide serves as a way to help keep consistency in clarity, formatting, and layout between all articles of the osu!wiki.
 
 All English articles should use plain English.
-Your word choice should be able to explain such things in layman's terms (basically ask yourself, "If the reader is reading this, would they have to lookup any of the words?").
+Your word choice should be able to explain such things in layman's terms.
```

Comment:
is this better?: `Your word choice should explain the topic that the article aims to explain thoroughly in layman's terms.`